src/marked.js

```diff
@@ -0,0 +1,10 @@
+import { updateStorage, todoList } from './savelist.js';
```

Comment:
- [OPTIONAL] Kindly adjust the styling of the footer to make the `clear all completed` button more visible.

Reply:
Thank you Zeena, yes, you are right.
thanks for contributing to my professionalism